requirements-dev.txt

```
@@ -26,7 +26,7 @@ mypy>=1.5.0
 ninja
 torchvision
 psutil
-cucim-cu12; platform_system == "Linux" and python_version >= "3.9" and python_version <= "3.10"
+#cucim-cu12; platform_system == "Linux" and python_version >= "3.9" and python_version <= "3.10"
```

I suppose it was a mistaken comment.
```suggestion
cucim-cu12; platform_system == "Linux" and python_version >= "3.9" and python_version <= "3.10"
```

---

Exactly, my CUDA version was too old to install it, I forgot to revert it to the original, thank you.